### PR TITLE
Add DoIT.life as Impossible

### PR DESCRIPTION
**Pull request by sue-woo**
No response to deletion request email after 5 weeks.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -3117,6 +3117,17 @@
             "docusign.com"
         ]
     },
+    
+    {
```
Comment (tupaschoal) [inline _data/sites.json:3121]:
```suggestion

    {
```

File: _data/sites.json [hunk 2]
```diff
@@ -3117,6 +3117,17 @@
...
+        "name": "Do IT",
+        "url": "https://doit.life/privacy-policy",
+        "notes": "I emailed data@doit.life but have not received a response after 5 weeks.",
```
Comment (tupaschoal) [inline _data/sites.json:3124]:
```suggestion
        "notes": "E-mailing them is a deadend even after waiting for several weeks.",
```

**Comment by sue-woo** [comment]
Hmm doesn't seem to like the space in Do It
`Keys must be in alphanumeric order. docusign needs to come after do it`

**Review by tupaschoal — COMMENTED** [review]
@sue-woo the CI complains that the list is not sorted, I believe due to the whitespace.

Can you move the new entry to just before `docker`, please?

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**